mypy/errors.py

```diff
@@ -370,12 +370,14 @@ def report_internal_error(err: Exception, file: str, line: int) -> None:
         pdb.post_mortem(sys.exc_info()[2])
     tb = traceback.extract_stack()[:-2]
     tb2 = traceback.extract_tb(sys.exc_info()[2])
-    print('Traceback (most recent call last):')
+    print('Traceback (most recent call last):', file=sys.stderr)
     for s in traceback.format_list(tb + tb2):
-        print(s.rstrip('\n'))
-    print('{}: {}'.format(type(err).__name__, err))
-    print('\n*** INTERNAL ERROR ***')
+        print(s.rstrip('\n'), file=sys.stderr)
+    print('{}: {}'.format(type(err).__name__, err), file=sys.stderr)
+    print('\n*** INTERNAL ERROR ***', file=sys.stderr)
```

Comment:
I'd argue that it is intentional that the traceback is written to stdout but the final two messages are written to stderr. Though I don't feel strongly.